### PR TITLE
Config for AIThinker ESP32-G Gateway

### PR DESCRIPTION
**Pull request by mrbubble62**
Board comes with u.fl antenna connection, Ethernet, USB serial, 4 LEDs, 5v and 12v power input, reset and boot buttons.
### https://eadalabs.com/esp32g-smart-ethernet-gateway/

### CODE REVIEW

**Comment by nagyrobi** [comment]
Also on ali: HUF 9,606.24  12%OFF | ESP32-G WiFi+BLE smart gateway Ethernet ESP32-G can be used with WiFi Bluetooth module
https://a.aliexpress.com/_mqiuXSU

File: ESP32-G.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  clk_mode: GPIO17_OUT
+  phy_addr: 1
+  power_pin: GPIO5
+
+api:
+logger:
+ota:
+
+dashboard_import:
+  package_import_url: github://esphome/bluetooth-proxies/esp32g.yaml@main
```
Comment (jesserockz) [inline ESP32-G.yaml:29]: This filename does not match
```suggestion
  package_import_url: github://esphome/bluetooth-proxies/ESP32-G.yaml@main
```

**Review by jesserockz — CHANGES_REQUESTED** [review]
You will need to add this to the static index page and the workflows files so people can actually install from the website.

**Comment by mrbubble62** [comment]
Not sure how to test the static page, I get an error  "Failed to download manifest" when I click on the Connect button for any of the boards.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ jobs:
           - wt32-eth01.yaml
           - gl-s10.yaml
           - lilygo-t-eth-poe.yaml
+          - esp32-g.yaml
```
Comment (balloob) [inline .github/workflows/ci.yml:19]: Add ai thinker to the filename

File: ESP32-G.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  clk_mode: GPIO17_OUT
+  phy_addr: 1
+  power_pin: GPIO5
+
+api:
+logger:
+ota:
+
+dashboard_import:
+  package_import_url: github://esphome/bluetooth-proxies/ESP32-G.yaml@main
```
Comment (balloob) [inline ESP32-G.yaml:29]: Capitalization needs to match the filename.

**Comment by balloob** [comment]
The PCB image is a) not used and b) 2MB

**Pull request closed without merging**